docs/developers_guide.rst

```diff
@@ -119,6 +120,11 @@ The simplest way to debug ORA2 code is with PDB - Python's built in debugger.
 One caveat: Since ORA2 has code that executes either in studio or lms context, one must be attached to the corret shell 
 in order for the breakpoints to be hit.
 
+Debugging JavaScript
+------------------
+Adding ``debugger`` to the JavaScript should work for ``development build`` and ``development server``.
```

Comment:
I'd even expand to say, look for this folder path in the dev tools. It took me a while to find on my own...

Reply:
Could you add a comment with where they are? I can't find them

Reply:
Where would you suggest me changing? I am not sure where else to put it.